packages/core/smoke-test/smoke-test.js

```diff
@@ -90,7 +91,7 @@ const downloadRepoZip = async workdir => {
   const zipPath = path.join(workdir, 'repo.zip');
   const dest = fs.createWriteStream(zipPath);
 
-  return await new Promise((resolve, reject) => {
```

Comment:
Had a lint failure here - no need to `return await` since the result is already getting `await/.then`'d